src/Lachain.Core/Blockchain/VM/ExternalHandler.cs

```diff
@@ -308,6 +308,14 @@ public static void Handle_Env_WriteEvent(int signatureOffset, int valueOffset, i
             };
             frame.InvocationContext.Snapshot.Events.AddEvent(ev);
         }
+        
+        public static void Handler_Env_GetAddress(int resultOffset)
+        {
+            var frame = VirtualMachine.ExecutionFrames.Peek() as WasmExecutionFrame
+                        ?? throw new InvalidOperationException("Cannot call KECCAK256 outside wasm frame");
+            var result = (frame.CurrentAddress).ToBytes();
+            SafeCopyToMemory(frame.Memory, result, resultOffset);
```

Comment:
Add returned value check?
```
if (!ret)
    throw new InvalidContractException("Bad call to (get_address)");
```

Reply:
No, as we are not returning any values here, and execution frame will always be having the `currentAddress`. So, essentially this check is not required .